src/main/java/jarvis/commands/UserList.java

```diff
@@ -0,0 +1,146 @@
+package jarvis.commands;
+
+import jarvis.Formatter;
+import jarvis.display.DisplayMessages;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+
+public class UserList extends Task {
+    private static ArrayList<Task> userList= new ArrayList<>(); //list of Tasks
```

Comment:
According to the comment, this attribute represents a list of tasks. You might want to rename the attribute to say, `tasksList` instead. This avoid the needs to add an additional comment since the name is already self-explanatory.